packages/optimizers/terser/src/TerserOptimizer.js

```diff
@@ -19,6 +19,7 @@ export default new Optimizer({
       );
     }
 
+    //TODO bundle.filePath should be path to entry asset
```

Comment:
The optimizer needs access to the entry asset to load the config

Reply:
@wbinnssmith ?

Reply:
Oh, seeing this now.

As far as I know, `loadConfig` looks upward the filesystem from the path, so it's not exclusively the dist directory. That seems fine to me.

Why does the optimizer need the entry asset?

Reply:
> As far as I know, loadConfig looks upward the filesystem from the path, so it's not exclusively the dist directory. That seems fine to me.

What about this case?
```
project
 - dist
 - src
   - .terserrc
   - index.js
```

And the dist directory could also be outside of the project dir

> Why does the optimizer need the entry asset?

The search for the config needs to start from *some* source asset. The entry asset seems reasonable.

Reply:
Yeah, I kind of like having the terser config relative to the dist directory. That way, you could even have multiple terser configs across targets.

I go back on forth on whether it makes sense to store terser config with the sources like that, but I can see it being surprising if we don't pick it up. @devongovett?

Reply:
> I kind of like having the terser config relative to the dist directory

This doesn't really work with the the split input/output fs PR...

Reply:
@wbinnssmith The config is loaded via `options.inputFS.readFile`.
"relative to the dist directory" isn't possible anymore